modules/dnn/test/test_layers_1d.cpp

```diff
@@ -307,4 +307,38 @@ INSTANTIATE_TEST_CASE_P(/*nothting*/, Layer_Split_Test,
                             std::vector<int>({4, 5})
 ));
 
+// WORKING FOR 1D
+typedef testing::TestWithParam<tuple<std::vector<int>>> Layer_Permute_Test;
+TEST_P(Layer_Permute_Test, Accuracy_01D)
+{
+    LayerParams lp;
+    lp.type = "Permute";
+    lp.name = "PermuteLayer";
+
+    int order[] = {0}; // Since it's a 0D tensor, the order remains [0]
```

Comment:
How it works for `{1, 4}` case? Should be `{0, 1}` in that case

Reply:
even for `{}`, `{0}` `oder = [0, 1]` works.

Reply:
But does `order = [0]` works for `{1, 4}`?

Reply:
yes